apps/services/schemas.py

```diff
@@ -20,6 +20,12 @@ class CreateUserRequestIn(BaseInSchema):
 
 class CreateUserRequestOut(BaseOutSchema):
     user_request_id: int
+    created_at: datetime
```

Comment:
i'm not sure that you implemented it right

Reply:
It's working